packages/authentication/src/__tests__/fixtures/helper.ts

```diff
@@ -104,9 +104,13 @@ export function createBearerAuthorizationHeaderValue(
 export function createUserProfile(user: User): UserProfile {
   const userProfile = {id: '', name: ''};
 
-  if (user && user.id) userProfile.id = user.id;
-  if (user && user.firstname && user.surname)
-    userProfile.name = `${user.firstname} ${user.surname}`;
+  if (user.id) userProfile.id = user.id;
+
+  let userName = '';
```

Comment:
nitpick: maybe simplify the code here as 
```ts
const userName = `${user.firstName} ${user.lastName}`
```

Reply:
Maybe also call `.trim()` in case there's no first name so there's no empty space before?

Reply:
@jannyHou 
>nitpick: maybe simplify the code here as
> const userName = `${user.firstName} ${user.lastName}` 
I am not sure I understand.
The code I used to have would only set  the `name` property if `both` first and surname were available (see Nora's comment in other PR).
So this new code will:
- set the `name` field to `firstName` if firstName is available and lastName is not.
- set the `name` field to `lastName` if lastName is available and firstName is not.
- set the `name` field to `firstName lastName` is both are available.

Reply:
@emonddr I think 

```ts
  if (user.firstName) userName = user.firstName;
  if (user.lastName)
    userName = user.firstName ? `${userName} ${user.lastName}` : user.lastName;
```

does

> set the name field to firstName if firstName is available and lastName is not.
> set the name field to lastName if lastName is available and firstName is not.
> set the name field to firstName lastName is both are available.

which is the same thing as what 
```ts
const userName = `${user.firstName} ${user.lastName}`
```
does...

But +1 for @nabdelgadir 's comment above^

Reply:
@nabdelgadir @jannyHou , if firstName is undefined, won't `${user.firstName} ${user.lastName}` evaluate to 'undefined someLastName' ?

Reply:
```
/ const user = {
//   firstName: 'boo',
//   lastName : 'hoo',
// };

const user = {  
  lastName : 'hoo',
};

console.log(`${user.firstName} ${user.lastName}`);

```

This prints out:

```
undefined hoo
```

Reply:
> This prints out:
> `undefined hoo`

Ah I didn't know that. Then your code makes perfect sense. LGTM 👍

Reply:
LGTM to me too